source/_docs/email.md

```diff
@@ -140,9 +140,18 @@ No, mail logs are not available for download and we do not recommend using the l
 ### What ports are recommended by Pantheon?
 Configuring mail to use port 25, 465 or 587 is strongly discouraged because they attract SPAM activities. Make sure that your service provider allows traffic on a port other than those mentioned and that you have correctly configured your site to use that port.
```

Comment:
This is.  just.  wrong!
> Configuring mail to use port 25, 465 or 587 is strongly discouraged because they attract SPAM activities. Make sure that your service provider allows traffic on a port other than those mentioned and that you have correctly configured your site to use that port.

I propose the following replacement:

> Pantheon strongly encourages using ports other than 25, 465 or 587 to send email because those ports are often blocked by service providers as an anti-spam measure. Here’s a list of popular email providers and the alternate ports which Pantheon recommends:

Reply:
@ari-gold Please review my proposed changes with an eye toward improving the doc.  I offer these proposed changes in that spirit, but I don't feel strongly that my proposed versions should be accepted.  Your call.

Reply:
Are we going to document the name of the A and AAAA records somewhere so interested parties can use them?   dig mail-relay.pantheon.io any

Reply:
We could, but what would they use them for? They should send mail via the php `mail()` function, so I don't think there is a common use for these. If there is a use case that necessitates using this, then we should document it in the context of that use case.